AWSS3Tests/AWSS3TestHelper.m

```diff
@@ -0,0 +1,141 @@
+//
+// Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// A copy of the License is located at
+//
+// http://aws.amazon.com/apache2.0
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+//
+
+#import "AWSS3TestHelper.h"
+#define AWS_TEST_BAH_INSTEAD true
```

Comment:
Is this still necessary now that this region has launched? (Not just the assignment, but the entire `AWS_TEST_BAH_INSTEAD` flag.)

Reply:
Removed